fastai/data/block.py

```diff
@@ -77,7 +77,9 @@ def _zip(x): return L(x).zip()
 @docs
 @funcs_kwargs
 class DataBlock():
-    "Generic container to quickly build `Datasets` and `DataLoaders`"
```

Comment:
Please don't change the formatting of existing fastai code, and when you add code please try to follow fastai standards. Our standard for docstrings is to use a single line and a single pair of double quotes.